controllers/pulp/api.go

```diff
@@ -907,6 +907,9 @@ func (r *PulpReconciler) pulpServerSecret(ctx context.Context, m *repomanagerv1a
 
 	// Handling user facing URLs
 	rootUrl := "http://" + m.Name + "-web-svc." + m.Namespace + ".svc.cluster.local:24880"
+	if strings.ToLower(m.Spec.IngressType) == "ingress" {
+		rootUrl = "https://" + m.Spec.IngressHost
```

Comment:
IngressHost is defined as optional. If users do not set a value for it, will Pulp still be accessible?
https://github.com/pulp/pulp-operator/blob/9dfd086ede337953eabaf8b08be9be368f2e2dfc/api/v1alpha1/pulp_types.go#L113-L115

Reply:
I wasn't unsure about this one, it depends on `ingress_type=Ingress` so I was thinking if I didn't make it optional it would require it for other ingress types